shared/src/packages/drivers/libs/types/service/driver-payload.type.ts

```diff
@@ -0,0 +1,16 @@
+import { type UserEntityT } from '~/packages/users/libs/types/types.js';
+
+import { type DriverEntity } from '../driver-entity.type.js';
+
+type DriverPayload = {
```

Comment:
also let's rewrite it, it's quite difficult to read now this type
I propose something like
type UserCommonDetails - pick necessary fields from User
type DriverDetails - pick driver license number
type DriverDto = UserCommonDetails & DriverDetails
type DriverCreateUpdateRequestBody - DriverDto & { password: string }